### PR TITLE
limit refresh on embeddings panel when clearing selection

### PR DESCRIPTION
**Pull request by manivoxel51**
## What changes are proposed in this pull request?

watch proposed fix below. 

- [x] Don't cause refresh the embedding panel when clearing embeddibngs
- [x] Confirm fix

[Updated] Proposed fix 

https://github.com/voxel51/fiftyone/assets/109545780/b9814216-65fc-4830-a283-8173b4757858

## How is this patch tested? If it is not, please explain why.

(Details)

### Is this a user-facing change that should be mentioned in the release notes?

-   [ ] No. You can skip the rest of this section.
-   [ ] Yes. Give a description of this change to be included in the release
        notes for FiftyOne users.

(Details in 1-2 sentences. You can just refer to another PR with a description
if this PR is part of a larger change.)

### What areas of FiftyOne does this PR affect?

-   [ ] App: FiftyOne application changes
-   [ ] Build: Build and test infrastructure changes
-   [ ] Core: Core `fiftyone` Python library changes
-   [ ] Documentation: FiftyOne documentation changes
-   [ ] Other

### CODE REVIEW

**Comment by brimoor** [comment]
@manivoxel51 sorry looks like we had a miscommunication. The sample grid *does* need to refresh every time the selection is changed (it always needs to stay in sync with the extended view, which changes whenever embeddings selection changes). The specific thing that is unnecessary is that the embeddings panel should not re-render (which can take awhile and causes things like zoom/pan to be reset unnecessarily). 

On `develop` right now, changing from `n` to `m != n` points selected does not cause the embeddings panel to fully reset, but going from `n` to `0` does cause it to refresh. The latter thing is the specific thing we want to avoid.

**Comment by manivoxel51** [comment]
> @manivoxel51 sorry looks like we had a miscommunication. The sample grid _does_ need to refresh every time the selection is changed (it always needs to stay in sync with the extended view, which changes whenever embeddings selection changes). The specific thing that is unnecessary is that the embeddings panel should not re-render (which can take awhile and causes things like zoom/pan to be reset unnecessarily).
> 
> On `develop` right now, changing from `n` to `m != n` points selected does not cause the embeddings panel to fully reset (good), but going from `n` to `0` does cause it to refresh (bad). The latter thing is the specific thing we want to avoid.

cool makes sense! that should be doable

**Comment by benjaminpkane** [comment]
The change is working for me 🚀 . I see that clearing the filters was added (or moved) to the panel close effect. So this means sidebar filters are reset when the panel is closed (see recording)? Is that expected behavior?

https://github.com/voxel51/fiftyone/assets/19821840/bfc98a57-5c71-452d-abac-a0c931933884

File: app/packages/embeddings/src/usePlotSelection.tsx [hunk 1]
```diff
@@ -38,7 +37,6 @@ export function usePlotSelection() {
     resetExtendedSelection();
     setPlotSelection(null);
     setSelectedSamples(new Set());
-    setFilters({});
```
Comment (manivoxel51) [inline app/packages/embeddings/src/usePlotSelection.tsx:41]: this

**Review by benjaminpkane — APPROVED** [review]
LGTM 🚀 I am fine with skipping e2e as this fixes a pending (loading) state. May be hard (or overkill) to test that at this stage

**Pull request merged**